xcode/Subconscious/Shared/Services/DataService.swift

```diff
@@ -492,10 +494,9 @@ struct DataService {
         )
 
         switch address {
-        case .public(let slug):
-            let slashlink = slug.toSlashlink()
+        case .public(let slashlink):
             do {
-                let memo = try noosphere.read(slashlink: slashlink)
+                let memo = try noosphere.read(slashlink: slashlink.description)
```

Comment:
Reading from real Slashlink means we can read `@petname/foo` content when Noosphere supports it.